Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-Article [XIX] (Amendment 19 - Women's Suffrage Rights)
-The right of citizens of the United States to vote shall not be denied or abridged by the United States or by any State on account of sex. affects 15
+Article V (Article 5 - The Enumeration of the Inherent Rights and Responsibilities of the People)
+The following are recognized as the Rights of the People, which are not granted by any earthly Authority, but inhere in their being as conscious, sentient lives, and which carry with them Responsibilities which cannot be enforced by any earthly Authority.
 
-Congress shall have power to enforce this article by appropriate legislation.   ratified #19
+1: It is the Right of all People to be secure in their person and property from aggression, coercion, unwarranted search and seizure, and theft; and to defend their selves and their property from such aggression, coercion, unwarranted search or seizure, or theft; and to possess, and to acquire by peaceful means, arms and other tools to accomplish that defense; and to dispose of their person and property in any manner they wish; and the Responsibility of all People to avoid, to the best of their ability, causing damage to another Person or that Person's Property without that Person's assent, and to exercise reasonable restraint in defense of their own Person or Property.
 
-Article [XX] (Amendment 20 - Terms of President, Vice President, Members of Congress: Presidential Vacancy)
-1: The terms of the President and Vice President shall end at noon on the 20th day of January, and the terms of Senators and Representatives at noon on the 3d day of January, of the years in which such terms would have ended if this article had not been ratified; and the terms of their successors shall then begin.   affects 5
+2: It is the Right of all People to associate or decline to associate with other People; and to form contracts; and to assemble in public places, and in private places with the assent of the owner of the place; and to engage in commerce, or refuse to engage in commerce, as fits their individual needs and desires; and the Responsibility of all People to ensure that their associations work toward the betterment of all associated, without causing harm to those not associated.
```

Comment:
Given that people own property, they have a right to trade, and to decline to trade. This extension of the understanding of property is explicitly laid out to remind all that they are under no obligation to associate with others, and only should do so when they desire to.

This stands as a rebuke to bills such as the US Affordable Care Act, which required citizens to purchase a product regardless of their own desires.